src/util/strencodings.h

```diff
@@ -305,4 +322,17 @@ std::string ToUpper(const std::string& str);
  */
 std::string Capitalize(std::string str);
 
+/**
+ * Parse a string with suffix unit [k|K|m|M|g|G|t|T] .
```

Comment:
nit: extra white space

```suggestion
 * Parse a string with suffix unit [k|K|m|M|g|G|t|T].
```

Reply:
fixed